official/benchmark/benchmark_uploader.py

```diff
@@ -72,19 +61,17 @@ def upload_benchmark_run(self, dataset_name, table_name, run_id):
         the data will be uploaded.
       run_id: string, a unique ID that will be attached to the data, usually
         this is a UUID4 format.
+      run_json: dict, the JSON data that contains the benchmark run info.
```

Comment:
The naming of this is confusing-- is it JSON or a python dict? If the latter, why is it called json?

Reply:
I think we should absolutely not refer to dicts as JSON. Some dicts can be converted to JSON, but I tend to think that json in the variable names implies that the variable refers to the serialized string.